framework/src/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -430,15 +430,29 @@ public boolean hasAnnotation(Class<? extends Annotation> a) {
     }
 
     /**
+     * Returns the actual effective annotation mirror used to annotate this type,
+     * whose Class equals the passed annoClass if one exists, null otherwise.
+     *
+     * @param annoClass annotation class
+     * @return the annotation mirror for anno
+     */
```

Comment:
I agree that we should add more see annotations to javadoc, in particular in this class.
However, that seems like a separate, larger, clean-up that also explains the relationship between plain, effective, explicit, and all the other kinds of getter methods.